src/components/ComposerAttachments.vue

```diff
@@ -134,11 +134,16 @@ export default {
 			return done
 		},
 		onAddCloudAttachment() {
-			const picker = getFilePickerBuilder(t('mail', 'Choose a file to add as attachment')).build()
+			const picker = getFilePickerBuilder(t('mail', 'Choose a file to add as attachment')).setMultiSelect(true).build()
 
 			return picker
 				.pick(t('mail', 'Choose a file to add as attachment'))
-				.then((path) => this.emitNewAttachment(this.fileNameToAttachment(path)))
+				.then((path) =>
+				{
+					let values = [];
+					path.forEach(p => {values.push(this.fileNameToAttachment(p))});
```

Comment:
are you familiar with `Array.map`? I'd prefer that over an iteration with a manual push to this temporary array.

Reply:
and I would prefer if we could change the `emitNewAttachment` to `emitNewAttachments` that takes an array so all paths go through the same helper :)

Reply:
Hey there, thanks for replying.
This is what I did for production environment on our company installation.
I'll get into your suggestions in my freetime (btw. good suggestions)